http/src/lib.rs

```diff
@@ -438,23 +438,24 @@ fn serve<M: jsonrpc::Metadata, S: jsonrpc::Middleware<M>>(
 		let handle = handle.clone();
 		bind_result.and_then(move |(listener, local_addr)| {
 			let allowed_hosts = server_utils::hosts::update(allowed_hosts, &local_addr);
-
+			
 			let http = {
-				let mut http = server::Http::new();
-				http.keep_alive(keep_alive);
-				http.sleep_on_errors(true);
-				http
+				let mut h = server::Http::new();
+				h.keep_alive(keep_alive);
```

Comment:
can we get `sleep_on_errors` back? I believe parity-1.11 doesn't have the latest hyper which has this enabled by default.